Makefile

```diff
@@ -203,8 +203,8 @@ gen: deps gen/grpc gen/openapi gen/openapi-server gen/converter
 
 .PHONY: lint
 lint:
-	${GOLANGCI_LINT} run main.go
-	${GOLANGCI_LINT} run cmd/... internal/... ./pkg/...
+	${GOLANGCI_LINT} run main.go  --timeout 3m
+	${GOLANGCI_LINT} run cmd/... internal/... ./pkg/...  --timeout 3m
```

Comment:
not sure why this was needed, but okay.

Reply:
In s390x environment , it was failing without timeout . By giving timeout option of 3 min , it waits for the operation to complete during the interval & is working fine post that